pkg/dist/service_types.yml

```diff
@@ -6601,23 +6481,30 @@ pg:
         - is_deprecated: true
           deprecation_notice: This value is deprecated.
           value: "11"
-        - value: "12"
+        - is_deprecated: true
+          deprecation_notice: This value is deprecated.
+          value: "12"
         - value: "13"
         - value: "14"
         - value: "15"
         - value: "16"
-        - value: "17"
     pgaudit:
+      is_deprecated: true
+      deprecation_notice: This property is deprecated.
```

Comment:
We may still want to keep pgaudit if its released soon, same applies to pg_qualstats

Reply:
I think it's not deprecated in the final diff. Probably happened because of switching back and force from old to the new schema.